library/timing.c

```diff
@@ -443,7 +485,34 @@ int mbedtls_timing_self_test( int verbose )
         mbedtls_printf( "passed\n" );
 
     if( verbose != 0 )
-        mbedtls_printf( "  TIMING test #2 (set/get_delay        ): " );
+        mbedtls_printf( "  TIMING test #2 (set_alarm / get_timer in uSec.): " );
+
+    for( secs = 1; secs <= 3; secs++ )
```

Comment:
This new self-test reverts a change we made in the self-tests because they were often failing on our CI. We can't take in the new self-test as it is, and there are now unit tests instead. But we might provisionally take in the feature without tests running on CI and add the tests after the release.